Formula/mesa.rb

```diff
@@ -61,10 +83,39 @@ def install
     ENV.prepend_path "PATH", "#{venv_root}/bin"
 
     mkdir "build" do
-      system "meson", *std_meson_args, "..", "-Db_ndebug=true"
+      args = %w[
+        -Db_ndebug=true
+      ]
```

Comment:
```suggestion
      args = ["-Db_ndebug=true"]
```
We don't need to split this up over several lines for one entry.